app/controllers/users.js

```diff
@@ -60,11 +60,39 @@ exports.signin = (req, res, next) => {
   const user = req.body ? req.body : {};
   emailValidation(req.body.email);
   userService.signin(user).then((signedInUser) => {
-    const token = sessionManager.encode({ username: signedInUser.email });
+    sessionManager.generateTokenAccess(signedInUser).then((tokenAccess) => {
+      res.status(200);
+      res.set('renewId', tokenAccess.renewId);
+      res.set('expirationDate', tokenAccess.expirationDate);
+      res.set(sessionManager.HEADER_NAME, tokenAccess.token);
```

Comment:
This way of sending it back is fine, but you also can send it in the response body, like: `res.send(tokenAccess)`.
Usually this kind of thing are agreed with the frontend app as how they will expect this, so in here as we don't really have a frontend app you can do it as you prefer.